conf/salt/project/django/run_with_db.sh

```diff
@@ -0,0 +1,5 @@
+# Shell script to set up necessary environment variables and run a PostgreSQL command
+export PGHOST="{{ pillar['env']['DB_HOST'] }}"
+export PGUSER="{{ pillar['project_name'] }}_{{ pillar['environment'] }}"
+export PGPASSWORD="{{ pillar['secrets']['DB_PASSWORD'] }}"
+exec "$@"
```

Comment:
Sorry about the ambiguity.

I would like the script to set PGDATABASE, keeping in mind that it's just a default if no database is mentioned on the command line, and it seems in line with what we're doing with the other common parameters to Postgres commands. 

But I might be missing something; I'm still not sure I follow your argument against setting PGDATABASE. Is there an example where having PGDATABASE set would break something?